### PR TITLE
feat: german SVNR (also called RVNR) support

### PR DESCRIPTION
**Pull request by sunghyukpark**
- Add validation for de.svnr

### CODE REVIEW

**Comment by sunghyukpark** [comment]
@koblas Relates to https://github.com/koblas/stdnum-js/issues/14. Can you take a look the pr?

File: src/ad/nrt.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ const impl: Validator = {
 
     const mid = v.substr(1, v.length - 2);
 
-    if (!strings.isalpha(v[0]) || !strings.isalpha(v[v.length - 1])) {
+    if (!strings.isalphanumeric(v[0]) || !strings.isalphanumeric(v[v.length - 1])) {
```
Comment (koblas) [inline src/ad/nrt.ts:61]: Re-reading the NRT specification it's `A-NNNNNN-A`, so the validators should be `strings.isalpha()` in both cases.  Unless you have an alternate reference.

File: src/de/svnr.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  '28',
+  '29',
+  '38',
+  '39',
+  '40',
+  '80',
+  '81',
+  '82',
+  '89'
+].concat([...Array(79 - 42 + 1).keys()].map(x => (x + 42).toString())) // '42'-'79'
```
Comment (koblas) [inline src/de/svnr.ts:50]: In both of these two cases you shouldn't need the `[...` spread operators, since they're returning an array anyways.

**Review by koblas — CHANGES_REQUESTED** [review]
Looks pretty close, can you please address the noted issues.  Also cleanup the PR title to match the semantic rules mentioned at the bottom of the readme.

File: src/de/svnr.ts (hunk 2 above)
Comment (sunghyukpark) [reply src/de/svnr.ts:50]: ^ adding `.keys()` returns iterator (`(method) Array<any>.keys(): IterableIterator<number>`), and without spread operator, I'm getting below type error. 
`Operator '+' cannot be applied to types 'IterableIterator<number>' and 'number'.`

Comment (sunghyukpark) [reply src/de/svnr.ts:50]: @koblas Can you approve the pr if no other changes are required?

**Review by koblas — APPROVED** [review]

**Comment by koblas** [comment]
Thanks for the contribution!

**Pull request merged**